shopsdk/orderphotos.py

```diff
@@ -0,0 +1,111 @@
+import json as a
+import pickle as b
+import os as c
+import pwinty_py3 as d
+import datadog as am
+import utilsnotbyme as i
+import ntplib as ar
+import pyminizip as aq
+import digitalcontentsender as au
+def pwinty(k, l, m, n, o, p, q, r, s, t, u, v, w, ao, ap, x):
+  z='rb'
+  with open("./config.json", z) as f:
+    g = a.load(f)
+    ab=g["pwinty_production"]
+    ac=g["pwinty_send_logs"]
+    ad=g["pwinty_use_pickle"]
+    ae=g["pwinty_execute_all"]
+  if ab==True and ac==True and g["pwinty_load_options_file"] and ad==True:
+    if c.path.exists("options.dat"):
+      h = open(options.dat, z)
+      options = b.load(h)
+    else:
+      bb = "Uh-oh!"
+      i.log("options.dat does not exist!! %s" % bb, "CRITICAL")
```

Comment:
Why is `options.dat` needed outside of the config?

Reply:
That is the datadog options JSON, the docs stated it needed it?

Reply:
Could we instead integrate it within the existing config, if necessary? I'm not quite sure where datadog is being called whatsoever.

Reply:
I didn't use it too much, but there are a few console messages I could probably use Sentry and datadog for. Datadog was just kinda there to tell you that the script was ran. (along with ntp time it was ran)

Reply:
Actually, I can probably only use Sentry, datadog is going to be really only used in the digicam print channel scripts I made.

Reply:
Yeah, I won't need datadog anymore. :)